composeApp/src/commonMain/kotlin/world/respect/app/app/RespectAsyncImage.kt

```diff
@@ -0,0 +1,23 @@
+package world.respect.app.app
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import coil3.compose.AsyncImage
+import com.eygraber.uri.Uri
+
+@Composable
+fun RespectAsyncImage(
+    uri: String,
+    contentDescription: String,
+    contentScale: ContentScale,
+    modifier: Modifier
+) {
+    AsyncImage(
+        model = remember { Uri.parse(uri) },
```

Comment:
This use of remember needs to use a key - otherwise when the Uri changes nothing will happen